RickMorty/app/src/main/res/layout/fragment_details.xml

```diff
@@ -24,28 +32,167 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:scaleType="centerCrop"
-            android:src="@drawable/rick" />
-
-        <TextView
-            android:id="@+id/detailsStatus"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:text="Alive"
-            android:textSize="18sp"
-            android:textColor="#000000"
-            android:layout_marginLeft="10dp"/>
-
+            android:downloadUrl="@{selectedCharacter.image}"/>
```

Comment:
örneğin databinding variable ı olarak viewmodel verdiğin durumda burası şu şekilde olacak:

`android:downloadUrl="@{viewModel.selectedCharacter.image}"`